contrib/babelfishpg_tsql/src/rolecmds.c

```diff
@@ -658,6 +658,9 @@ user_name(PG_FUNCTION_ARGS)
 						 &is_null);
 	user = pstrdup(TextDatumGetCString(datum));
 
+	systable_endscan(scan);
+	table_close(bbf_authid_user_ext_rel, RowExclusiveLock);
+
```

Comment:
can we add a test case for this where we can validate that no locks are held on this table